puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/hammer-cli-master-release.groovy

```diff
@@ -0,0 +1,6 @@
+def git_url = 'https://github.com/theforeman/hammer-cli.git'
+def git_ref = 'master'
+def package_name = 'rubygem-hammer_cli'
+def project_name = 'hammer-cli'
+def build_rpm = true
+def build_deb = false
```

Comment:
We do ship hammer in debs, but we don't build nightly debs for hammer?

Reply:
Correct -- https://github.com/theforeman/foreman-infra/pull/933/files#diff-fd91d137d8bc077e688a82182d8e3e29L38